src/test/java/pageObjects/LoginPageTests.java

```diff
@@ -4,13 +4,70 @@
 import org.testng.annotations.Test;
 
 public class LoginPageTests extends BaseTest {
-
     @Test
-    public void signIn() {
-        loginPage.signIn();
+    public void successfulSignInAsSuperAdmin() throws InterruptedException {
+        loginPage.successfulSignInAsSuperAdmin();
         String URL = driver.getCurrentUrl();
-
-        Assert.assertEquals(URL, "https://www.google.com/" );
-        System.out.println("PASSED");
+        Assert.assertEquals(URL, "https://zenistock.zenitech.local/dashboard" );
+    }
+    @Test
+    public void successfulSignInAsAdmin() throws InterruptedException {
+        loginPage.successfulSignInAsAdmin();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "https://zenistock.zenitech.local/dashboard" );
+    }
+    @Test
+    public void successfulSignInAsUser() throws InterruptedException {
+        loginPage.successfulSignInAsUser();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "https://zenistock.zenitech.local/dashboard" );
+    }
+    @Test
+    public void logInNoUsername(){
+        loginPage.logInNoUsername();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL,"https://zenistock.zenitech.local/login");
+    }
+    @Test
+    public void logInNoPassword(){
+        loginPage.logInNoPassword();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "https://zenistock.zenitech.local/login");
+    }
+    @Test
+    public void logInNoPasswordNoUsername(){
+        loginPage.logInNoPasswordNoUsername();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL,"https://zenistock.zenitech.local/login");
+    }
+    @Test
+    public void logInWithInvalidUsername(){
+        loginPage.logInWithInvalidUsername();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "https://zenistock.zenitech.local/login");
+    }
+    @Test
+    public void logInWithInvalidPassword(){
+        loginPage.logInWithInvalidPassword();
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "https://zenistock.zenitech.local/login");
+    }
```

Comment:
for these tests, can we also check if the field is required or if there is any invalid credentials message displayed?